src/test/kotlin/lotto/model/LottoMachineTest.kt

```diff
@@ -0,0 +1,11 @@
+package lotto.model
+
+import org.junit.jupiter.api.Test
+
+class LottoMachineTest {
+    @Test
+    fun `구매 금액만큼 로또를 만든다`() {
```

Comment:
```suggestion
    fun `8000원으로 로또 8개를 만든다`() {
```

테스트 메서드는 보다 구체적으로 네이밍하면 테스트 코드를 보는 동료 개발자가 더 이해하기 쉬울 것이에요 😉